obsreport/obsreport.go

```diff
@@ -35,6 +37,24 @@ var (
 	okStatus = trace.Status{Code: trace.StatusCodeOK}
 )
 
+// SetParentLink tries to retrieve a span from sideCtx and if one exists
+// sets its SpanID, TraceID as a link in the span provided. It returns
+// true only if it retrieved a parent span from the context.
+func SetParentLink(sideCtx context.Context, span *trace.Span) bool {
+	parentSpanFromRPC := trace.FromContext(sideCtx)
```

Comment:
What does RPC signify here?

Reply:
Good point, the code comes from the observability package but it is the right moment to improve it.